__tests__/app.test.js

```diff
@@ -19,8 +20,13 @@ describe("GET /api/topics", () => {
       .then((response) => {
         expect(response.status).toBe(200);
         expect(Array.isArray(response.body.topic)).toBe(true);
-        expect(response.body.topic.length).toBeGreaterThan(0);
-        expect(typeof response.body.topic[0]).toBe("object");
+        expect(response.body.topic.length).toBe(3);
+        const expectedProperties = ["slug", "description"];
+        response.body.topic.forEach((element) => {
+          expectedProperties.forEach((property) => {
+            expect(element).toHaveProperty(property);
+          });
```

Comment:
Well done for checking length. However, we want you to check that each object has a slug and description key with a string as the value for each.